include/irr/asset/IPipelineLayout.h

```diff
@@ -38,7 +39,7 @@ class IPipelineLayout
     const DescLayoutType* getDescriptorSetLayout(uint32_t _set) const { return m_descSetLayouts[_set]; }
     const core::refctd_dynamic_array<SPushConstantRange>* getPushConstantRanges() const { return m_pushConstantRanges.get(); }
 
-    core::smart_refctd_ptr<DescLayoutType> m_descSetLayouts[4];
+    std::array<core::smart_refctd_ptr<DescLayoutType>, 4> m_descSetLayouts;
```

Comment:
try not to ever use random integer literals, even if you know what they mean... constexpr is the shit.

Reply:
@Crisspl ?

Reply:
resolved